Added documentation for cat?sort and format. (#7619)
opensearch-project/documentation-website: _api-reference/cat/index.md

```
{% include copy-curl.html %}

> The response is an ASCII cat (`=^.^=`) and a list of operations.

Reviewer: ```suggestion
The response is an ASCII cat (`=^.^=`) and a list of operations:
```

```
=^.^=
/_cat/allocation
/_cat/segment_replication
/_cat/segment_replication/{index}
/_cat/shards
/_cat/shards/{index}
/_cat/cluster_manager
/_cat/nodes
/_cat/tasks
/_cat/indices
/_cat/indices/{index}
/_cat/segments
/_cat/segments/{index}
/_cat/count
/_cat/count/{index}
/_cat/recovery
/_cat/recovery/{index}
/_cat/health
/_cat/pending_tasks
/_cat/aliases
/_cat/aliases/{alias}
/_cat/thread_pool
/_cat/thread_pool/{thread_pools}
/_cat/plugins
/_cat/fielddata
/_cat/fielddata/{fields}
/_cat/nodeattrs
/_cat/repositories
/_cat/snapshots/{repository}
/_cat/templates
/_cat/pit_segments
/_cat/pit_segments/{pit_id}
```

## Optional query parameters

The root `_cat` API does not take any parameters, but individual APIs, such as `/_cat/nodes` accept the following query parameters.

Parameter | Description
:--- | :--- |
`v` |  Provides verbose output by adding headers to the columns. It also adds some formatting to help align each of the columns together. All examples in this section include the `v` parameter.
`help` | Lists the default and other available headers for a given operation.
`h`  |  Limits the output to specific headers.

> `format` |  Returns the result in JSON, YAML, CBOR, or Smile formats.

Reviewer: ```suggestion
`format` |  The format in which to return the result. Valid values are `json`, `yaml`, `cbor`, and `smile`.
```

GET _cat/<operation_name>?help
```

> For example, the aliases help contains all available headers.

Reviewer: ```suggestion
For example, to see the available headers for the CAT aliases operation, send the following request:
```

```json
GET _cat/aliases?help
```
{% include copy-curl.html %}

> ```

Reviewer: ````suggestion
The response contains the available headers:

```
````

GET _cat/<operation_name>?h=<header_name_1>,<header_name_2>&v
```

> For example, to limit aliases to only the alias name and index.

Reviewer: ```suggestion
For example, to limit aliases to only the alias name and index, send the following request:
```

```json
GET _cat/aliases?h=alias,index
```
{% include copy-curl.html %}

> ```

Reviewer: ````suggestion
The response contains the requested information:

```
````

.kibana .kibana_1
sample-alias1 sample-index-1
```

Typically, for any operation you can find out what headers are available using the `help` parameter, and then use the `h` parameter to limit the output to only the headers that you care about.

### Sort by a header

To sort the output by a header, use the `s` parameter:

> ```

Reviewer: ````suggestion
```json
````

GET _cat/<operation_name>?s=<header_name_1>,<header_name_2>
```

> For example, to sort aliases by alias then index.

Reviewer: ```suggestion
For example, to sort aliases by alias and then index, send the following request:
```

```json
GET _cat/aliases?s=i,a
```
{% include copy-curl.html %}

> ```

Reviewer: ````suggestion
The response contains the requested information:

```
````

sample-alias2 sample-index-1
sample-alias1 sample-index-2
```

### Retrieve data in JSON format

> By default the CAT APIs return `text/plain` data.

Reviewer: ```suggestion
By default, CAT APIs return data in `text/plain` format.
```

> To retrieve data in the JSON format, use the `format=json` parameter:

Reviewer: ```suggestion
To retrieve data in JSON format, use the `format=json` parameter:
```

> ```

Reviewer: ````suggestion
```json
````

GET _cat/<operation_name>?format=json
```

> For example, to retrieve aliases in JSON format.

Reviewer: ```suggestion
For example, to retrieve aliases in JSON format, send the following request:
```

```json
GET _cat/aliases?format=json
```
{% include copy-curl.html %}

> ```json

Reviewer: ````suggestion
The response contains data in JSON format:

```json
````
